solr/server/scripts/cloud-scripts/snapshotscli.sh

```diff
@@ -11,7 +11,7 @@ run_solr_snapshot_tool() {
     log4j_config="file:${scriptDir}/../../resources/log4j2-console.xml"
   fi
   PATH=${JAVA_HOME}/bin:${PATH} ${JVM} ${ZKCLI_JVM_FLAGS} -Dlog4j.configurationFile=${log4j_config} \
-  -classpath "${solrLibPath}" org.apache.solr.core.snapshots.SolrSnapshotsTool "$@" 2> /dev/null
+  -classpath "${solrLibPath}" org.apache.solr.hdfs.snapshots.SolrSnapshotsTool "$@" 2> /dev/null
```

Comment:
Do we need to add modules/hdfs/... to `solrLibPath` for this tool to work? Why did the SolrSnapshotsTool need to move to hdfs module in the first plae?

Reply:
`SolrSnapshotsTool` has a bunch of HDFS specific dependencies in it. I looked at trying to separate it a little when moving HDFS to a module, but it wasn't simple. I didn't find any reference to this tool in the ref guide when I checked so I'm not sure how much it is actually used.

Reply:
The solrLibPath var is set like this
```bash
solrLibPath="${SOLR_LIB_PATH:-${scriptDir}/../../solr-webapp/webapp/WEB-INF/lib/*:${scriptDir}/../../lib/ext/*}"
```
I think `../../../modules/hdfs/lib/solr-hdfs*` is missing there?

Reply:
Addressed in feec4cbdf31583684d44d1fd78956c6533c28156

Reply:
The only hdfs specific dependency I see is Path... Is the commons-cli a transative dependency there? In general I'm not sure how useful this tool is now that the backup-restore functionality is really good, but I guess that's for a separate ticket.

Reply:
So its not just a compile time dependency on `Path` - if you read the code there are tons of references to HDFS and distcp. Basically for snapshots to work you need a shared filesystem and HDFS is one of the only ones that works semi easily. But yea agree about backup-restore and if snapshots still is needed.